lib/phobos/listener.rb

```diff
@@ -18,13 +18,14 @@ def initialize(handler:, group_id:, topic:, min_bytes: nil, max_wait_time: nil,
                    delivery: 'batch', max_bytes_per_partition: DEFAULT_MAX_BYTES_PER_PARTITION,
                    session_timeout: nil, offset_commit_interval: nil,
                    heartbeat_interval: nil, offset_commit_threshold: nil,
-                   offset_retention_time: nil)
+                   offset_retention_time: nil, max_retries: -1)
       @id = SecureRandom.hex[0...6]
       @handler_class = handler
       @group_id = group_id
       @topic = topic
       @backoff = backoff
       @delivery = delivery.to_s
+      @max_retries = max_retries.to_i
```

Comment:
if someone specified a wrong number in max_retries like string, this max_retries will be set to 0. And the retry logic above will not run as expected (but just do more retry with creating client). Seems harmless, but may need to consider.